src/components/AlbumCard.js

```diff
@@ -1,14 +1,24 @@
 import "./AlbumCard.css";
+import SongList from "./SongList";
+import { useState } from "react";
 
 // function has to create cards with the props: title, author and album cover
-export default function AlbumCard({ title, artist, albumCover }) {
+export default function AlbumCard({ title, artist, albumCover, songs }) {
+  const [displaySongs, setDisplaySongs] = useState(false);
   return (
     <article className="album-card">
-      <img className="album-card-img" src={albumCover} alt={title} />
+      <button
+        className="album-card-tracks"
+        onClick={() => setDisplaySongs(!displaySongs)}
+        aria-label="Click here to toggle the tracks list"
+      >
+        <img className="album-card-img" src={albumCover} alt={title} />
+      </button>
       <div className="album-card-content">
-        <p className="album-card-title">{title}</p>
-        <p className="album-card-artist">{artist}</p>
+        <p className="album-card-title">Title: {title}</p>
+        <p className="album-card-artist">Artist: {artist}</p>
       </div>
+      {displaySongs === true && <SongList songs={songs} />}
```

Comment:
you could shorten this but this solution is good too: 
```suggestion
      {displaySongs && <SongList songs={songs} />}
```